pandas/core/arrays/datetimes.py

```diff
@@ -514,7 +514,7 @@ def _unbox_scalar(self, value) -> np.datetime64:
             return value.as_unit(self.unit).asm8
 
     def _scalar_from_string(self, value) -> Timestamp | NaTType:
-        return Timestamp(value, tz=self.tz)
+        return Timestamp(value, tz=self.tz)  # .as_unit(self._unit)
```

Comment:
Does this mean `as_unit` will need to be called eventually?

Reply:
nope, removed